implementations/elixir/ockam/ockam/lib/ockam/transport/tcp/default_wrapper.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Ockam.Transport.TCP.DefaultWrapper do
+  @moduledoc """
+  Default implementation of a TCP wrapper behaviour.
+  """
+  @behaviour Ockam.Transport.TCP.Wrapper
+
+  @impl true
+  def wrap_tcp_call(fun) do
+    fun.()
+  end
```

Comment:
If we decided to use the MFA syntax, I think this could literally be a `Kernel.apply` call instead:
```suggestion
  def wrap_tcp_call(module, func, args \\ []) when is_list(args) do
    apply(module, func, args)
  end
```